selftests/check.py

```diff
@@ -496,8 +496,9 @@ def main():
     if args.features:
         features = []
         for suite in suites:
-            for variants in suite.config['run.dict_variants']:
-                features.append(variants['namespace'])
+            if suite.config.get('run.dict_variants') is not None:
+                for variants in suite.config['run.dict_variants']:
+                    features.append(variants['namespace'])
```

Comment:
We are assuming that all configs are there, because there was registered with a default value. So, if users are not setting a new value, a default value should be there.

I tried to avoid having checks like this in the code. Maybe I'm missing something, but this brings me a "yellow flag" that we should investigate why 'run.dict_variants' is not being populated with the default value.

Not sure if makes sense.

Reply:
That absolutely makes sense. Thanks for the review, @beraldoleal.